content/scripts/test_get_frontmatter_key.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
```

Comment:
[dust] I get that this is the expected output, but I still found it a bit confusing before looking at the tests, any way we could try and make this a tad clearer?
<img width="532" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/2387e375-8360-4710-9d9d-d75eb703fdc2">

Reply:
Yeah... I didn't love that either, much as I don't love our `console.log()` outputs when we run Jest. I've addressed this in a revised commit that buffers (`-b`) the output, so it shouldn't display anymore. Thanks for the feedback :)